docs/examples/api-walkthrough/app/src/main/java/com/couchbase/apiwalkthrough/MainActivity.java

```diff
@@ -41,36 +43,59 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // create database
         DatabaseConfiguration config = new DatabaseConfiguration(getApplicationContext());
-        final Database database = new Database("my-database", config);
+        config.setConflictResolver(new LocalWins());
+        Database database = null;
+        try {
+            database = new Database("my-database", config);
+        } catch (CouchbaseLiteException e) {
+            e.printStackTrace();
```

Comment:
In case of Exception is thrown, the following code which refers database will cause NPE. Better to return here.